scripts/create-action-plan-selectors/create-action-plan-selectors.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import sys
+
+import requests
+from sqlalchemy import create_engine
+from structlog import wrap_logger
+
+from config import Config
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_casetypeoverrides():
+    logger.debug('Retrieving casetypeoverrides')
+    engine = create_engine(Config.DATABASE_URI)
+    connection = engine.connect()
+
+    sql = "SELECT o.sampleunittypefk, o.actionplanid, c.exerciseref, c.survey_uuid " \
+          "FROM collectionexercise.casetypeoverride o " \
+          "INNER JOIN collectionexercise.collectionexercise c ON o.exercisefk = c.exercisepk"
+
+    trans = connection.begin()
+    response = connection.execute(sql)
+    trans.commit()
+    logging.debug('Successfully retrieved casetypeoverrides')
+    return [row for row in response]
+
+
+def get_survey_ref(survey_id):
+    logger.debug('Retrieving survey ref', survey_id=survey_id)
+    url = f'{Config.SURVEY_URL}/surveys/{survey_id}'
+    response = requests.get(url=url, auth=Config.SURVEY_AUTH)
+    response.raise_for_status()
+    survey_ref = response.json()['surveyRef']
+    logger.debug('Successfully retrieved surveyRef', survey_id=survey_id, survey_ref=survey_ref)
+    return survey_ref
+
+
+def update_action_plan_selectors(casetypeoverride):
+    action_plan_id = casetypeoverride['actionplanid']
+    exercise_ref = casetypeoverride['exerciseref']
+    sample_unit_type = casetypeoverride["sampleunittypefk"]
```

Comment:
single quotes for consistency, also these 3 things `actionplanid/exerciseref/sampleunittypefk` are always going to be present?